core/node/consistency_checker/src/lib.rs

```diff
@@ -177,10 +177,11 @@ impl LocalL1BatchCommitData {
         let commit_tx_hash = storage
             .eth_sender_dal()
             .get_confirmed_tx_hash_by_eth_tx_id(commit_tx_id as u32)
-            .await?
-            .with_context(|| {
-                format!("Commit tx hash not found in the database for tx id {commit_tx_id}")
-            })?;
+            .await?;
+
+        let Some(commit_tx_hash) = commit_tx_hash else {
+            return Ok(None);
+        };
```

Comment:
why have you changed it?

Reply:
`commiy_tx_id` is from `get_eth_commit_tx_id` it does not check if transaction is confirmed, but `get_confirmed_tx_hash_by_eth_tx_id` only returns for confirmed transactions (fastfinalized/finalized). The consistency_checker was crashing here.